lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,51 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {import('postcss-value-parser').Node[]} params
+ * @returns {Array<string>}
+ */
+function listImportConditions(params) {
+	if (!params.length) return [];
+
+	/** @type {Array<Array<import('postcss-value-parser').Node>>} */
+	let media = [];
+	/** @type {Array<import('postcss-value-parser').Node>} */
+	let lastMediaQuery = [];
+
+	for (let i = 0; i < params.length; i++) {
+		const param = params[i];
+
+		if (!param) break;
+
+		// remove whitespace to get a consistent key
+		if (param.type === 'space') {
+			continue;
+		}
+
+		// remove comments to get a consistent key
+		if (param.type === 'comment') {
+			continue;
+		}
+
+		if (param.type === 'div' && param.value === ',') {
+			media.push(lastMediaQuery);
+			lastMediaQuery = [];
```

Comment:
suggestion

since all `if` branches `continue` maybe it would be clearer to just put an additional `if ((i + 1) === params.length)` before l108 so that you can get rid of this reset

pretty minor, you could put `params.length` in a const